manifest.json

```diff
@@ -6,22 +6,22 @@
         "en": "Open Source Webmail software",
         "fr": "Webmail Open Source"
     },
+    "version": "1.3.6~ynh2",
     "url": "https://roundcube.net/",
     "license": "GPL-3",
```

Comment:
GPL-3.0-or-later or  GPL-3.0-only

Reply:
Indeed, but will be fixed by https://github.com/YunoHost-Apps/roundcube_ynh/pull/50

Reply:
Ok